catalog/plugin/catalog-plugin-defaultsecurityattributevalues/src/test/java/org/codice/alliance/catalog/plugin/defaultsecurity/DefaultSecurityAttributeValuesPluginTest.java

```diff
@@ -259,12 +258,12 @@ private static SystemHighAttributes createTestSystemHighAttributes() {
 
   private static Map<String, String> createTestInitialConfiguration() {
     final Map<String, String> configuration = new HashMap<>();
-    configuration.put(CLASSIFICATION_CONFIGURATION_KEY, "classification");
-    configuration.put(RELEASABILITY_CONFIGURATION_KEY, "releasableTo");
-    configuration.put(CODEWORDS_CONFIGURATION_KEY, "sciControls");
-    configuration.put(DISSEMINATION_CONTROLS_CONFIGURATION_KEY, "disseminationControls");
-    configuration.put(OTHER_DISSEMINATION_CONTROLS_CONFIGURATION_KEY, "otherDisseminationControls");
-    configuration.put(OWNER_PRODUCER_CONFIGURATION_KEY, "ownerProducer");
+    configuration.put(Security.CLASSIFICATION, "classification");
```

Comment:
✏️ Should probably add a negative test to ensure that an incorrectly formatted mapping (missing `=` or with more than one `=`) is ignored.

Reply:
Added negative test.